test/PackageManagement.Test/BuildIntegratedTests.cs

```diff
@@ -228,6 +228,68 @@ public class BuildIntegratedTests
         }
 
         [Fact]
```

Comment:
Do we have a test doing update-package (update all) to the build integrated project?